crates/nu-protocol/src/signature.rs

```diff
@@ -319,6 +319,8 @@ pub struct Signature {
     pub is_filter: bool,
     pub creates_scope: bool,
     pub allows_unknown_args: bool,
+    /// TODO: Use a proper type here
+    pub complete: Option<Option<DeclId>>,
```

Comment:
Another enum type named differently from the one used in Argument?

Reply:
If not an enum, then at least a doc comment explaining how this works